src/services/player/playerChapters.tsx

```diff
@@ -75,6 +75,7 @@ export const enrichChapterDataForPlayer = (chapters: any[], duration: number) =>
     }
   }
 
+  // TODO: Delete this block?
```

Comment:
This `enrichedChaptersFinal` doesn't seem to be used at all and can likely be removed.

Reply:
hmm not sure what i was going for there. will remove it.